server/models/Notification.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'user',
+  },
+  type: {
+    type: String,
+    enum: ['request', 'accept', 'decline'],
+  },
+  title: {
+    type: String,
+  },
+  description: {
+    type: String,
+    maxlength: 300,
+  },
+  read: {
+    type: Boolean,
+    default: false,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+  context: {
+    type: Object,
+  }
```

Comment:
Good one! Mongoose schemas have a `timestamps` option that tells Mongoose to automatically manage `createdAt` and `updatedAt` properties on your documents. Check it out [here](https://mongoosejs.com/docs/guide.html#timestamps)